src/core/lv_group.c

```diff
@@ -6,12 +6,11 @@
 /*********************
  *      INCLUDES
  *********************/
-#include <stddef.h>
-
 #include "lv_group.h"
 #include "../core/lv_obj.h"
 #include "../core/lv_global.h"
 #include "../indev/lv_indev.h"
+#include LV_STDDEF_INCLUDE
```

Comment:
How about adding lv_stddef.h similar to lv_string.h etc.

Reply:
What do we need from `stddef`? I think we use only `NULL` from there.